src/package.xml

```diff
@@ -1,125 +1,50 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<package name="helloworldUncertainty" displayName="Example demonstrating how to use iterations" version="1.1.0" url="https://apexrms.github.io/helloworldEnhanced/">	
-	<transformers>
+<package name="helloworldUncertainty" displayName="Hello World Uncertainty" description="Example demonstrating how to use iterations" version="2.0.0" url="https://apexrms.github.io/helloworldEnhanced/">
+
+	<!--Scenario Datasheets-->
+
+	<!--Run Control-->
+	<dataSheet name="RunControl" displayName="Run Control" isSingleRow="True">
```

Comment:
Replace this run control datasheet with:
`<dataSheet name="RunControl" displayName="Run Control" isRunControl="True" timestepTitle="Timestep" showMinIteration="False">
	</dataSheet>`